code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -4006,3 +4006,12 @@ datum
 			mix_phrase = "The mixture comes together slowly. It doesn't seem like it wants to be here."
 			required_reagents = list("poor_cement" = 1, "silicon_dioxide" = 5, "water" = 1)
 			result_amount = 7
+
+		hemotoxin
+			name = "hemotoxin"
+			id = "hemotoxin"
+			result = "hemotoxin"
+			required_reagents = list("oil" = 1, "neurotoxin" = 1, "acid" = 1)
+			result_amount = 1
```

Comment:
There a balance/thematic/chemistry reason for the yield? Or just scale welding fluid usage?

Reply:
hemotoxin and neurotoxin are common in snake venom, so i tought making one reaction change one snake venom to another would be interesting, oil i put there so chemists can make a small amount with just what they have on lab, but to make larger quantities would require more oil manufacturing.

Reply:
I was wondering about the: 3 units in -> 1 unit out.  2/3 of the solution disappears.
Often times that is done some realistic reactions that releases a byproduct (solid/liquid/gas) we don't care about or if some aspect of the reaction requires a catalyst.

There isn't anything necessarily wrong with it, just curious.

Reply:
i could add a byproduct... any ideas?